app/resources/forms.py

```diff
@@ -40,3 +41,14 @@ class ResourceForm(Form):
         Length(5, 5)
     ])
     submit = SubmitField('Add Resource')
+
+
+class ReviewForm(Form):
+    rating = IntegerField('Rating', validators=[
+        InputRequired(),
+        NumberRange(1, 5)
```

Comment:
if this 1-5, should the the comment on the model be changed? https://github.com/hack4impact/women-veterans-rock/pull/26/files#diff-095dfff0d906b2f540bfb0ca9f75a63aR24